bindings/go/osvdev/osvdev.go

```diff
@@ -81,15 +85,15 @@ func (c *OSVClient) GetVulnByID(ctx context.Context, id string) (*osvschema.Vuln
 // been retrieved are returned.
 //
 // See if next_page_token field in the response is fully filled out to determine if there are extra pages remaining
-func (c *OSVClient) QueryBatch(ctx context.Context, queries []*Query) (*BatchedResponse, error) {
+func (c *OSVClient) QueryBatch(ctx context.Context, queries []*api.Query) (*api.BatchVulnerabilityList, error) {
 	// API has a limit of how many queries are in one batch
 	queryChunks := chunkBy(queries, MaxQueriesPerQueryBatchRequest)
-	totalOsvRespBatched := make([][]MinimalResponse, len(queryChunks))
+	totalOsvRespBatched := make([][]*api.VulnerabilityList, len(queryChunks))
 
 	g, errGrpCtx := errgroup.WithContext(ctx)
 	g.SetLimit(c.Config.MaxConcurrentBatchRequests)
 	for batchIndex, queries := range queryChunks {
-		requestBytes, err := json.Marshal(BatchedQuery{Queries: queries})
+		requestBytes, err := protojson.Marshal(&api.BatchQuery{Queries: queries})
```

Comment:
We probably need to `UseProtoNames` or add the json_name to the protos like https://github.com/ossf/osv-schema/pull/442 (for e.g. `next_page_token`) for all these API translations.